tests/Feature/Subscription.php

```diff
@@ -120,6 +121,27 @@ public function testUpdateSubscriptionPlan(callable $subscriptionFactory)
         $this->assertEquals(Arr::dissoc($previous, ['plan', 'price']), Arr::dissoc($next, ['plan', 'price']));
     }
 
+    /**
+     * @group integration
+     * @dataProvider subscriptionProvider
+     *
+     * @return void
+     */
+    public function testUpdateSubscriptionPlanToPlanWithDifferentBillingCycle(callable $subscriptionFactory)
```

Comment:
We can probably add a few more tests, but I think this covers us for our use case?

Reply:
Let's include this during our pair jam.